### PR TITLE
added jasmine-spec-reporter

### PR DESCRIPTION
**Pull request by Enriquesuarezgtte**
screen-reporter

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,6 +25,7 @@
   },
   "homepage": "https://github.com/Enriquesuarezgtte/protractor-workshop-2018#readme",
   "dependencies": {
+    "jasmine-spec-reporter": "^4.2.1",
```
Comment (leonleo997) [inline package.json]: is this a dependency or a dev dependency?

File: protractor/helpers/reporter.ts [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import { SpecReporter } from "jasmine-spec-reporter";
+
```
Comment (leonleo997) [inline protractor/helpers/reporter.ts]: delete whitespace

File: protractor/helpers/reporter.ts [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+
```
Comment (leonleo997) [inline protractor/helpers/reporter.ts]: delete whitespace

File: package.json (hunk 1 above)
Comment (Enriquesuarezgtte) [reply package.json]: my mistake. it should be a dev dependency

File: protractor/helpers/reporter.ts (hunk 2 above)
Comment (Enriquesuarezgtte) [reply protractor/helpers/reporter.ts]: done

File: protractor/helpers/reporter.ts (hunk 3 above)
Comment (Enriquesuarezgtte) [reply protractor/helpers/reporter.ts]: done

File: protractor/helpers/reporter.ts [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import { SpecReporter } from "jasmine-spec-reporter";
+
+export let reporter = ()=>{
```
Comment (leonleo997) [inline protractor/helpers/reporter.ts]: did you indent the code?

**Pull request merged**